app/controllers/visns_controller.rb

```diff
@@ -73,6 +73,10 @@ def show
 
   def load_facilities_show_rows
     @facilities = VaFacility.get_by_visn(@visn).select(:common_name, :id, :official_station_name, :slug, :station_number, :fy17_parent_station_complexity_level)
+    @clinical_resource_hubs = ClinicalResourceHub.cached_clinical_resource_hubs
+    visn_id = Visn.find_by(number: params["number"].to_i).id
+    @clinical_resource_hubs = @clinical_resource_hubs.where(visn_id: visn_id)
```

Comment:
Can we call the second `@clinical_resource_hubs` something else? Maybe like `@visn_clinical_resource_hub`? We're only trying to return one `ClinicalResourceHub`, so I don't think we need it to be plural, plus it's overwriting the instance variable name from line `76`.

Reply:
Hey Josh - good catch... i'm just using @facilities += Array(ClinicalResourceHub.find_by(visn_id: visn_id)) on line 78 so we don't even need that other variable... and the var on line 76 doesn't get overwritten.